whole-tale/ngx-dashboard
# feat: use collections for ext data if catalog is enabled

## Problem

Currently "add data" modal only works with registered data. However, some WT deployments use data managed directly by girder. There's no easy way to add that data to Tales in that case.

## Approach
Allows to use native Girder data in Tales if `wholetale.enable_data_catalog` is set to True

## How to Test

1. Checkout this branch locally, rebuild the dashboard
2. Login to the WholeTale Dashboard
3. Create a Tale
4. Register some external data, verify that 'add data' modal works.
5. Navigate to Girder. In Wholetale plugin settings, enable Data Catalog. Create some collections and populate them with data.
6. Go back to dashboard, verify that 'add data' modal now works on collections
